keyboards/crkbd/keymaps/like_jis/keymap.c

```diff
@@ -0,0 +1,234 @@
+#include "crkbd.h"
```

Comment:
Could you remove (most of) the includes here and replace them with just `#include QMK_KEYBOARD_H`?